app/views/teacher_training_adviser/steps/_accept_privacy_policy.html.erb

```diff
@@ -1,11 +1,7 @@
 <% policy_id = session[:privacy_policy_id].presence || GetIntoTeachingApiClient::PrivacyPoliciesApi.new.get_latest_privacy_policy.id %>
 
 <%= f.govuk_check_boxes_fieldset :accepted_policy_id, legend: { text: "Read and accept the privacy policy"} do %>
-  <p>Our privacy notice explains how we use your personal data. It is important that you have read this notice before continuing.</p>
-
-  <div class="govuk-form-group">
-    <%= link_to "Privacy policy", privacy_policy_path(:id => policy_id) %>
-  </div>
+  <p>Read our <%= link_to("privacy policy", privacy_policy_path, target: "blank") %> before you continue. It explains how we use your personal data.</p>
```

Comment:
@ethax-ross I note the code in the removals also had a reference to `policy_id`, do we need to reference that here or not?

Reply:
Ah, yes well spotted - that ensures the policy they accept is the one that we have shown them (on the off chance it changes after they view it but before they submit the form). Updated.